android-components/components/feature/addons/src/main/res/values/strings.xml

```diff
@@ -173,6 +173,18 @@
     <string name="mozac_feature_addons_successfully_installed">Successfully installed %1$s</string>
     <!-- Text shown after failed to install an add-on. %1$s is the add-on name. -->
     <string name="mozac_feature_addons_failed_to_install">Failed to install %1$s</string>
+    <!-- Text shown when attempting to install an add-on and a network error happened. -->
+    <string name="mozac_feature_addons_failed_to_install_network_error">The add-on could not be downloaded because of a connection failure</string>
+    <!-- Text shown when attempting to install an add-on and the downloaded file is corrupted. -->
+    <string name="mozac_feature_addons_failed_to_install_corrupt_error">This add-on could not be installed because it appears to be corrupt</string>
+    <!-- Text shown when attempting to install an add-on and the downloaded hash file is the wrong one. -->
+    <string name="mozac_feature_addons_failed_to_install_hash_error">This add-on could not be installed because it does not match the expected</string>
```

Comment:
```suggestion
    <string name="mozac_feature_addons_failed_to_install_hash_error">This add-on could not be installed because it does not match the expected.</string>
```